js/components/frame.js

```diff
@@ -768,6 +770,9 @@ class Frame extends ImmutableComponent {
       if (this.doNotTrack) {
         this.webview.executeJavaScript('Navigator.prototype.__defineGetter__("doNotTrack", () => {return 1});')
       }
+      if (hack && hack.allowRunningInsecureContent) {
+        appActions.changeSiteSetting(this.origin, 'runInsecureContent', true)
```

Comment:
this still isn't going to work correctly because the site setting won't update in time. We should remove the siteHacks for insecure content and translate them to site setting defaults